src/Views/Form/defaultFormTemplate.php

```diff
@@ -1,30 +1,19 @@
 <?php
+/**
+ * Donation form view.
+ *
+ * @since 2.7.0
+ */
+use Give\TemplateSkinManager;
 use function Give\Helpers\Form\Template\Utils\Frontend\getFormId;
 
 $formId = getFormId();
-?>
-<!DOCTYPE html>
-<html <?php language_attributes(); ?>  style="margin-top: 0 !important;">
-	<head>
-		<meta charset="utf-8">
-		<title><?php echo apply_filters( 'the_title', get_post_field( 'post_title', $formId ) ); ?></title>
-		<?php
-		/**
-		 * Fire the action hook in header
-		 */
-		do_action( 'give_embed_head' );
-		?>
-	</head>
-	<body class="give-form-templates">
-		<?php
+$tm     = new TemplateSkinManager();
 
-		// Fetch the Give Form.
-		give_get_donation_form( [ 'id' => $formId ] );
+// Fetch the Give Form.
+ob_start();
```

Comment:
Is this never closed? Why is it here?

Reply:
I closed it below. Check under `SetBody` fn.

Reply:
Oh I see. That is a *perfect* example, of why functions should not just randomly write to the output buffer. Then we get to do weird stuff like this to capture output. 🤦

Reply:
resolved